package.json

```
@@ -17,8 +17,9 @@
   },
   "dependencies": {
     "@nuxtjs/style-resources": "^1.0.0",
-    "@spryker-vsf/api": "^0.1.0",
-    "@spryker-vsf/composables": "^0.1.0",
+    "@spryker-vsf/api": "^0.2.0-next.0",
+    "@spryker-vsf/catalog-search-suggestions": "^0.1.0-next.0",
+    "@spryker-vsf/composables": "^0.1.1-next.0",
```

Please update to the stable versions
```suggestion
    "@spryker-vsf/api": "^0.2.0",
    "@spryker-vsf/catalog-search-suggestions": "^0.1.0",
    "@spryker-vsf/composables": "^0.1.1",
```